docs/profile-service/api-contracts.md

```diff
@@ -0,0 +1,122 @@
+# Profile Service API Specifications and Contracts
+
+Profile Service is a custom backend set up by everyone in the RDS community. It includes 3 endpoints:
+
+1. `/health`
+2. `/profile`
+3. `/verify`
+
+## Endpoint: /health
+
+This is a `GET` endpoint that returns a message "Profile Service - Health Verified".
+
+- **Request**
+
+GET `/health`
+
+- **Response**
+
+HTTP/1.1 200 OK
+Content-Type: application/json
+
+{
+"message": "Profile Service - Health Verified"
+}
+
+## Endpoint: /profile
+
+This is a `GET` endpoint that returns a JSON object in the following format. It is a verified route, you need to verify the token sent in the header when a request is made to this route. It will be verified with the chain code provided when linked to RDS.
+
+- **Request**
+GET /profile
+Authorization: Bearer <JWT_TOKEN>
+
+- **Response**
+
+- Success (HTTP Status 200 OK):
+
+```json
+HTTP/1.1 200 OK
+Content-Type: application/json
+
+{
+    "first_name": "YOUR_FIRST_NAME",
+    "last_name": "YOUR_LAST_NAME",
+    "email": "YOUR_EMAIL_ID",
+    "phone": "YOUR_PHONE_NUMBER",
+    "yoe": YOUR_YEARS_OF_EXPERIENCE_AS_INTEGER,

```

Comment:
Any specific reason why this field is colored ?

Reply:
because it's direct text without quotes and in md I have mentioned it as json. If I do not mention json/any language it lints out.